package-system/AWSNativeSDK/README.md

```diff
@@ -0,0 +1,100 @@
+# Build New AWS Service Target in AWSNativeSDK
```

Comment:
I would add a line about the intent of the README.md, it jumps straight into adding a new AWS Service Target.

# Overview
This package defines and builds the [AWS C++ SDK]() libarires for O3DE. It builds all core libraries and an opinionated subset of AWS client libraries. To add a new AWS Service client (or target) please use the instructions below.

Reply:
will do